### PR TITLE
docs: maintainers meeting 2023-11-02

### PR DESCRIPTION
**Pull request by inigomarquinez**
## Description

Add maintainers meeting notes from 2023-11-02

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-18.12.1
+18
```
Comment (UlisesGascon) [inline .nvmrc:1]: This can be removed as https://github.com/onebeyond/maintainers/pull/153 is merged :)

File: docs/03-meetings/2023-11-02.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+title: "2023-11-02"
+slug: /meetings/2023-11-02
+---
+
+# One Beyond Open Source Maintainers Meeting 2023-11-02
+
+TODO: add YouTube embed video
+
+## Links 
+* **recording:** [Link](<to be added>)
```
Comment (UlisesGascon) [inline docs/03-meetings/2023-11-02.md:12]: We can remove references for the video recording for now due technical issues with the Google Account. 

```suggestion
## Links 
```

**Review by UlisesGascon — CHANGES_REQUESTED** [review]

File: .nvmrc (hunk 1 above)
Comment (UlisesGascon) [reply .nvmrc]: Solved in [5fd2534](https://github.com/onebeyond/maintainers/pull/152/commits/5fd2534350621df0b93e5a47470a3863cf9f8d8a)

Comment (inigomarquinez) [reply .nvmrc]: Thanks

**Review by UlisesGascon — APPROVED** [review]

**Pull request merged**